test/Spec/Integration.hs

```diff
@@ -79,6 +84,48 @@ test = do
                     ("Should be 2 UTxO at destination wallet, but request returned " <> show utxoCnt)
                     (utxoCnt == 2)
 
+getUtxos :: Contract () EmptySchema Text (Map TxOutRef ChainIndexTxOut)
+getUtxos = do
+  pkh <- Contract.ownPaymentPubKeyHash
+  Contract.logInfo @String $ printf "Own PKH: %s" (show pkh)
+  utxosAt $ pubKeyHashAddress pkh Nothing
+
+getUtxosThrowsErr :: Contract () EmptySchema Text (Map TxOutRef ChainIndexTxOut)
+getUtxosThrowsErr = do
+  pkh <- Contract.ownPaymentPubKeyHash
+  Contract.logInfo @String $ printf "Own PKH: %s" (show pkh)
+  utxos <- utxosAt $ pubKeyHashAddress pkh Nothing
+  Contract.throwError (mkMsg utxos)
+  where
+    mkMsg utxos =
+      pack $
+        mconcat
+          [ "This Error was thrown intentionally by Contract \n"
+          , "Debug: Own UTxOs: " <> show utxos <> "\n"
+          ]
+
+getUtxosThrowsEx :: Contract () EmptySchema Text (Map TxOutRef ChainIndexTxOut)
+getUtxosThrowsEx = do
```

Comment:
Maybe we can simplify this contract just to 
```
getUtxosThrowsEx = error "This Exception was thrown intentionally in Contract.\n"
```
(that UTxO data I used in examples in message was just to make sure some contract calls were really executed before error, but here it feels unnecessary for me, as we just checking that exception was caught as expected)

Reply:
And `getUtxosThrowsErr` can be just `Contract.throwError "This Error was thrown intentionally by Contract "` too